code/game/objects/items/plushes.dm

```diff
@@ -66,8 +141,196 @@
+		user.visible_message("<span class='notice'>[user] pronounces \the [Kisser] and \the [src] married! D'aw.</span>",
+									"<span class='notice'>You pronounce \the [Kisser] and \the [src] married!</span>")
+		new_partner(Kisser)
+		Kisser.new_partner(src)
+
+	//then comes a baby in a baby's carriage, or an adoption in an adoption's orphanage
+	else if(Kisser.partner == src && !plush_child)	//the one advancing does not take ownership of the child and we have a one child policy in the toyshop
+		user.visible_message("<span class='notice'>[user] is going to break \the [Kisser] and \the [src] by bashing them like that.</span>",
+									"<span class='notice'>\The [Kisser] passionately embraces \the [src] in your hands. Look away you perv!</span>")
+		plop(Kisser)
+		user.visible_message("<span class='notice'>Something drops at the feet of [user].</span>",
+							"<span class='notice'>The miracle of oh god did that just come out of \the [src]?!</span>")
+
+	//then comes protection, or abstinence if we are catholic
+	else if(Kisser.partner == src && plush_child)
+		user.visible_message("<span class='notice'>[user] makes \the [Kisser] nuzzle \the [src]!</span>",
+									"<span class='notice'>You make \the [Kisser] nuzzle \the [src]!</span>")
+
+	//then oh fuck something unexpected happened
+	else
+		user.show_message("<span class='warning'>\The [Kisser] and \the [src] don't know what to do with one another.</span>", 0)
+
+/obj/item/toy/plush/proc/heartbreak(obj/item/toy/plush/Brutus)
+	if(lover != Brutus)
+		return	//why are we considering someone we don't love?
+
+	scorned += Brutus
+	Brutus.scorned_by(src)
+	lover = null
+	Brutus.lover = null	//feeling's mutual
+	heartbroken = TRUE
+	mood_message = heartbroken_message
+
+	if(partner == Brutus)	//oh dear...
+		partner = null
+		Brutus.partner = null	//it'd be weird otherwise
+		vowbroken = TRUE
+		mood_message = vowbroken_message
+
+	update_desc()
+
+/obj/item/toy/plush/proc/scorned_by(obj/item/toy/plush/Outmoded)
+	scorned_by += Outmoded
+
+/obj/item/toy/plush/proc/new_lover(obj/item/toy/plush/Juliet)
+	if(lover == Juliet)
+		return	//nice try
+	lover = Juliet
+	mood_message = love_message
+	update_desc()
+
+	if(partner)	//who?
+		partner = null	//more like who cares
+
+/obj/item/toy/plush/proc/new_partner(obj/item/toy/plush/Apple_of_my_eye)
+	if(partner == Apple_of_my_eye)
+		return	//double marriage is just insecurity
+	if(lover != Apple_of_my_eye)
+		return	//union not born out of love will falter
+
+	partner = Apple_of_my_eye
+	mood_message = "\n\The [src] has a ring on \his finger! It says bound to my dear [partner]."
+	update_desc()
+
+/obj/item/toy/plush/proc/plop(obj/item/toy/plush/Daddy)
+	if(partner != Daddy)
+		return	//we do not have bastards in our toyshop
+
+	if(prob(50))	//it has my eyes
+		plush_child = new type(get_turf(loc))
+	else	//it has your eyes
+		plush_child = new Daddy.type(get_turf(loc))
+
+	plush_child.make_young(src, Daddy)
```

Comment:
set the mood message of the parents in the make_young proc rather than outside it

Reply:
i.e move the below code into the make young proc for each parent

Reply:
Good catch.